DatabaseHandler.py

```diff
@@ -0,0 +1,112 @@
+# General imports =================================================================================
+import json
+import multiprocessing as mp
+from pocketbase import Client
+from pocketbase.services.realtime_service import MessageData
+
+from br_threading.WorkQCommands import WorkQCmnd, WorkQCmnd_e
+
+
+# Class Definitions ===============================================================================
+class DatabaseHandler():
+    def __init__(self, db_thread_workq: mp.Queue):
+        """
+        Thread to handle the pocketbase database communication.
+        The Thread is subscribed to the CommandMessage
+        collection to wait for commands created in the front end.
+        The handler can also send telemetry data to the database
+        to be read by the front end.
+        """
+        DatabaseHandler.db_thread_workq = db_thread_workq
+        # DatabaseHandler.client = Client('http://192.168.0.70:8090')
+        DatabaseHandler.client = Client('http://192.168.0.130:8090')
+        # DatabaseHandler.client = Client('http://127.0.0.1:8090')
+
+        DatabaseHandler.client.collection('PlcCommands').subscribe(DatabaseHandler._handle_command_callback)
+
+        print("DB - thread started")
+
+    @staticmethod
+    def _handle_command_callback(document: MessageData):
+        """
+        Whenever a new entry is created in the CommandMessage
+        collection, this function is called to handle the
+        command and forward it to the serial port.
+
+        Args:
+            document (MessageData): the change notification from the database.
+        """
+
+        print(f"DB - Received {document.record.command} command")
+        message = WorkQCmnd(WorkQCmnd_e.COMMAND_FROM_DB, document.record.command)
+        DatabaseHandler.db_thread_workq.put(message)
+
+    @staticmethod
+    def write_plc_data(plc_data: bytes):
+        entry = {}
+        entry["plc_data"] = list(plc_data)
+
+        try:
+            DatabaseHandler.client.collection("PLC").create(entry)
+        except Exception as e:
+            print(f"failed to create a plc_data entry {e}")
```

Comment:
You might want to split this into 3 fields because we get data in 3 sets now

Reply:
split into a bunch for each individual value from the plc